packages/paste-website/src/pages/patterns/error-state/index.mdx

```diff
@@ -9,6 +9,7 @@ import {Alert} from '@twilio-paste/alert';
 import {Box} from '@twilio-paste/box';
 import {Card} from '@twilio-paste/card';
 import {Grid, Column} from '@twilio-paste/grid';
+import {Heading} from '@twilio-paste/heading';
```

Comment:
Console errors whilst running the site locally and testing pages. It'll complain about missing components in a mdx file but compile successfully because hoisting and bundling hides our inability to lint or type check an mdx page